components/drata/actions/update-vendor/update-vendor.mjs

```diff
@@ -0,0 +1,225 @@
+import drata from "../../drata.app.mjs";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/VendorsPublicController_updateVendor/";
+
+export default {
+  key: "drata-update-vendor",
+  name: "Update Vendor",
+  description: `Update a Vendor. [See docs here.](${docsLink})`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    drata,
+    vendorId: {
+      type: "integer",
+      label: "Vendor ID",
+      description: "The ID of the vendor",
```

Comment:
Can you implement async options for this based on the [list vendors](https://developers.drata.com/docs/openapi/reference/operation/VendorsPublicController_listVendors/) endpoint?